docs/running.md

```diff
@@ -11,4 +11,59 @@ To start the server in the foreground simply run:
 ```bash
 warnet
 ```
+
+## Running large networks
+
+When running a large number of containers on a single host machine (i.e. with the Docker interface), the system may run out of various resources.
+We recommend setting the following values in /etc/sysctl.conf:
+
+```sh
+# Increase ARP cache thresholds to improve network performance under high load
+# gc_thresh1 - Adjust to higher threshold to retain more ARP entries and avoid cache overflow
+net.ipv4.neigh.default.gc_thresh1 = 80000
+
+# gc_thresh2 - Set the soft threshold for garbage collection to initiate ARP entry clean up
+net.ipv4.neigh.default.gc_thresh2 = 90000
+
+# gc_thresh3 - Set the hard threshold beyond which the system will start to drop ARP entries
+net.ipv4.neigh.default.gc_thresh3 = 100000
+
+# Increase inotify watchers limit to allow more files to be monitored for changes
+# This is beneficial for applications like file sync services, IDEs or web development servers
+fs.inotify.max_user_watches = 100000
```

Comment:
I had to add this one too:
`fs.inotify.max_user_instances = 100000`

I was getting "too many open files" errors in the cadvisor tank

Reply:
added

Reply:
It also goes in /etc/sysctl.conf right?

Reply:
Yeah!

Reply:
noice. Want anything else in here?

Reply:
I guess not!